### DISCUSSION TITLE
Supporting Sphinx-Needs. What is possible?

### DISCUSSION THREAD

**Discussion by danwos** [useblocks/sphinx-needs#261] [Ideas]
*First draft to collect ideas.*
*Text and list will get updated during discussion.*

### Goal: Get things done for Sphinx-Needs by allowing support via different ways (time, money, skill, ...)

Getting PRs from the community and issues solved is awesome.
But there are user of Sphinx-Needs, who don't have the needed skills or time or motivation to solve an issue.
So lets use this discussion to see if there is a need for different kinds of supporting use.

### List of "support" services:

* https://issuehunt.io/  - Set bounties on issues
* https://opencollective.com/ - Collect money and spend it in a transparent way
* Github  ...

### Events:
* https://hacktoberfest.digitalocean.com/
* https://summerofcode.withgoogle.com/

**Comment by danieleades** [comment]
I've not got any personal experience with issuehunt, but I believe the general premise is to allow anyone to contribute bounties for tasks that are important to them, and anyone can collect that bounty by resolving it. It's a tool to both prioritise development important to enterprise users, and to solicit more input from the open source community.

As for the actual governance of who *decides* that a bounty has been earned, I have no idea. Will read up when I get a chance!

*If* my company adopts sphinx-needs as an internal tool I *may* be able to solicit bounties for issues which affect us, for example
